test/k4FWCoreTest/options/ExampleFunctionalReadNthEvent.py

```diff
@@ -0,0 +1,41 @@
+#
+# Copyright (c) 2014-2024 Key4hep-Project.
+#
+# This file is part of Key4hep.
+# See https://key4hep.github.io/key4hep-doc/ for further info.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# This is an example for reading from a file starting from a certain event and
+# tranforming so that later we can check the number of events in the output file
+
+from Gaudi.Configuration import INFO
+from Configurables import ExampleFunctionalTransformer
+from Configurables import EventDataSvc
+from k4FWCore import ApplicationMgr, IOSvc
+
+svc = IOSvc("IOSvc")
+svc.Input = "functional_producer.root"
+svc.Output = "functional_nth_event.root"
+svc.FirstEventEntry = 5
```

Comment:
Could it be changed to some other number so it's easier to reason? The input has 10 entries, the first entry is 5, the output has 5 entries. 6-4 or any other split would read better

Reply:
Done